### ISSUE TITLE
__set_name__ owner argument is the wrong class

### ISSUE THREAD

**Issue by Docopoper** [brython-dev/brython#1265]
The ``owner`` argument of ``__set_name__`` should be the class the attribute was defined in, not the class of the descriptor itself.

```
class TestDescriptor:
    def __get__(self, instance, owner):
        return self

    def __set_name__(self, owner, name):
        print(owner, name)

class TestObject:
    my_descriptor = TestDescriptor()
```
In this case the output should be ``<class '__main__.TestObject'> my_descriptor``
Brython is currently outputting ``<class '__main__.TestDescriptor'> my_descriptor``

**Comment by PierreQuentel** [comment]
I think that the commit that fixed issue #1264 gives the correct result. Can you check with the latest version ?

**Comment by PierreQuentel** [comment]
I am closing the issue, hoping it's fixed. If not, please reopen it.
